packages/yew/src/functional/hooks/use_force_update.rs

```diff
@@ -0,0 +1,84 @@
+use super::{Hook, HookContext};
+use crate::functional::ReRender;
+use std::fmt;
+
+/// A handle which can be used to force a re-render of the associated
+/// function component.
+#[derive(Clone)]
+pub struct UseForceUpdate {
+    trigger: ReRender,
+}
+
+impl fmt::Debug for UseForceUpdate {
+    fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
+        f.debug_struct("UseForceUpdate").finish()
+    }
+}
+
+impl UseForceUpdate {
+    /// Trigger an unconditional re-render of the associated function component
+    pub fn force_update(&self) {
+        (self.trigger)()
+    }
+}
+
+// #![feature(fn_traits)] // required nightly feature to make UseForceUpdate callable directly
+// impl Fn<()> for UseForceUpdate {
+//     extern "rust-call" fn call(&self, _args: ()) {
+//         self.force_update()
+//     }
+// }
```

Comment:
We should put this behind a `nightly` feature flag but that can be done in a separate PR